tests/e2e/tests/critical-path/database/clone-databases.e2e.ts

```diff
@@ -0,0 +1,112 @@
+import { rte } from '../../../helpers/constants';
+import { AddRedisDatabasePage, MyRedisDatabasePage } from '../../../pageObjects';
+import { commonUrl, ossClusterConfig, ossSentinelConfig, ossStandaloneConfig } from '../../../helpers/conf';
+import { acceptLicenseTerms } from '../../../helpers/database';
+import {
+    addNewOSSClusterDatabaseApi,
+    addNewStandaloneDatabaseApi,
+    deleteAllSentinelDatabasesApi,
+    deleteOSSClusterDatabaseApi,
+    deleteStandaloneDatabaseApi,
+    discoverSentinelDatabaseApi
+} from '../../../helpers/api/api-database';
+
+const addRedisDatabasePage = new AddRedisDatabasePage();
+const myRedisDatabasePage = new MyRedisDatabasePage();
+const newOssDatabaseAlias = 'cloned oss cluster';
+
+fixture `Clone databases`
+    .meta({ type: 'critical_path' })
+    .page(commonUrl);
+test
+    .before(async t => {
+        await acceptLicenseTerms();
+        await addNewStandaloneDatabaseApi(ossStandaloneConfig);
+        await t.eval(() => location.reload());
+    })
+    .after(async() => {
+        // Delete databases
+        const dbNumber = await myRedisDatabasePage.dbNameList.withExactText(ossStandaloneConfig.databaseName).count;
+        for (let i = 0; i < dbNumber; i++) {
+            await deleteStandaloneDatabaseApi(ossStandaloneConfig);
+        }
+    })
+    .meta({ rte: rte.standalone })('Verify that user can clone Standalone db', async t => {
```

Comment:
seems like all tests here use  .meta({ rte: rte.standalone })
so it can be moved at the top to the fixture meta

Reply:
or rte above can be changed to rte: rte.ossCluster and rte: rte.sentinel

Reply:
fixed, thanks